Source/iop/Iop_Dmacman.h

```diff
@@ -19,5 +20,9 @@ namespace Iop
 	private:
 		uint32 DmacRequest(CMIPS&, uint32, uint32, uint32, uint32, uint32);
 		void DmacTransfer(CMIPS&, uint32);
+		void DmacSetVal(CMIPS&, uint32, uint32);
```

Comment:
Could you change the function names here to also reflect the PS2SDK names? Thanks!

Reply:
Done

Reply:
Is the commit missing? The names are still the same.